website/content/docs/platform/k8s/consul-to-raft.mdx

```diff
@@ -0,0 +1,361 @@
+---
+layout: docs
+page_title: Migrate Consul to Raft storage
+description: >-
+  Guide to migration of Consul storage to Raft. 
+---
+
+# Migrate Consul to Raft storage
+
+This procedure assumes you have a Vault cluster deployed in a Kubernetes environment configured with Consul storage.  The storage migration can occur while leaving the Consul cluster intact.  A single change to the Consul cluster is a lock file written by Vault during the migration.
+
+This guide uses basic examples and default Vault configurations.  This guide is for illustrative purposes, and adaption to specific configurations relevant to your environment is still required.
+
+<Warning>
+
+Always back up your data before attempting migration! Although this is an offline operation and the risk is low, it is advisable to take a recent snapshot from your Consul cluster before proceeding.
+
+</Warning>
+
+## Summary
```

Comment:
This sounds like a brief overview of what this guide will walk through rather than a summary?

```suggestion
## Overview
```